README.md

```diff
@@ -126,6 +126,29 @@ MicroShift keeps all its state in its data-dir, which defaults to `/var/lib/micr
 
 When starting the MicroShift for the first time the Kubeconfig file is created. If you need it for another user or to use externally the kubeadmin's kubeconfig is placed at `/var/lib/microshift/resources/kubeadmin/kubeconfig`.
 
+### Pulling Container Image From Private Registries
+
+MicroShift may not have the pull secret for the registry that you are trying to use. For example, MicroShift does not have the pull secret for registry.redhat.io. In order to use this registry, there are several approaches. The first approach is to use podman login,
+```sh
+podman login registry.redhat.io
+```
+
+Once the podman login is complete, MicroShift will be able to pull images from this registry. This approach works across name spaces.
+
+This approach assumes podman is installed. This might not be true for all MicroShift environments. For example, if MicroShift is installed through RPM, CRI-O will be installed as dependency, but no podman. In this case, one can choose to install podman separately, or use other approaches described below. 
+
```

Comment:
* edit `as dependency` to `as a dependency` and  `but no podman` to `but not podman` 
* wdyt, is this more clear? `In this case, one can choose to install podman separate from the rpm and then login as above, or, to allow CRI-O access to private registries, create a pull secret as described below`